pkg/ibmvpc/vpc.go

```diff
@@ -17,6 +17,20 @@ func getNodeName(name, addr string) string {
 	return fmt.Sprintf("%s[%s]", name, addr)
 }
 
+type Region struct {
+	name string
+}
+
+func regionToStruct(regionName string, regionToStructMap map[string]*Region) *Region {
```

Comment:
1. rename it to getRegionByName()
2. I think it should be at parser.go. @adisos?

Reply:
yes, I also think so